docs/templates/README.md

```diff
@@ -0,0 +1,34 @@
+# Templates
+
+Find all provided documentation templates in this folder. Please note that the _italic text 
+and sentences_ should be removed. Feel free to add additional sections and subsections, however, make sure
+that at least the sections of the templates marked as "mandatory" are filled.
+
+## Decision Records
+
+[Link](decision-record.md) to template.
+
+Each decision record should be put in an appropriate folder that is following a naming pattern: 
+`YYYY-MM-DD-title-of-decision-record`. This should be located at the [decision record folder](../developer/decision-records) 
```

Comment:
nit: broken link. Pls ignore if we do not need this one here or we have some other ways to cover it from upstream.

Just adding the comment to highlight what is missing.

Reply:
removed sections specific to EDC, included templates